haystack/components/generators/chat/azure.py

```diff
@@ -149,6 +155,8 @@ def __init__(
             api_key=api_key.resolve_value() if api_key is not None else None,
             azure_ad_token=azure_ad_token.resolve_value() if azure_ad_token is not None else None,
             organization=organization,
+            timeout=self.timeout,
```

Comment:
Weird that timeout was being initialised but not actually being passed to `AzureOpenAI` earlier 😅